### DISCUSSION TITLE
[Frontend]Flutter/Dart

### DISCUSSION THREAD

**Discussion by CodeIdeal** [realworld-apps/realworld#851] [WIP Implementations]
I noticed that there is no Flutter implementation here, so I made one.
repo(WIP): https://github.com/CodeIdeal/realworld_flutter

**Comment by realshovanshah** [comment]
I actually already created a discussion a couple of days back regarding this. Was just waiting for confirmation before updating the repo.
Here's a link to the project: http://github.com/realshovanshah/realworld-flutter-app

**Reply by CodeIdeal** [reply]
Glad that someone is interested in flutter implementation. 
I was originally an Android developer, I learned flutter because my company's mobile development department is transitioning to the flutter technology stack. I am happy to work with experienced flutter engineers, because I can learn a lot of real production software experience while contributing.

**Reply by realshovanshah** [reply]
That's great, where can I contact you other that github? I tried emailing you at sms@codeideal.com (from your website) but it doesn't seem to exist.

**Reply by thisisamank** [reply]
Hey, @realshovanshah I am also interested in making a flutter implementation of the same. I want the UI of the conduit, do you have Figma file or something for the same?

**Reply by realshovanshah** [reply]
According to the [official specs](https://realworld-docs.netlify.app/docs/specs/mobile-specs/introduction#stylestemplates), using the Medium mobile app is recommended for creating your own implementation. 
You could also try running a mobile tagged example from the [official showcase list](https://codebase.show/projects/realworld?category=frontend).

**Comment by geromegrignon** [comment]
@skyliao1 here is a work in progress (based on your request #723)
